packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -561,6 +564,134 @@ describeWithDOM('mount', () => {
     });
   });
 
+  describeIf(is('>= 16.6'), 'Suspense & lazy', () => {
+    class DynamicComponent extends React.Component {
+      render() {
+        return (
+          <div>Dynamic Component</div>
+        );
+      }
+    }
+    class Fallback extends React.Component {
+      render() {
+        return (
+          <div>Fallback</div>
+        );
+      }
+    }
+    it('finds Suspense and its children when no lazy component', () => {
+      class Component extends React.Component {
+        render() {
+          return (
+            <div>test</div>
+          );
+        }
+      }
+      const SuspenseComponent = () => (
+        <Suspense fallback={Fallback}>
+          <Component />
+        </Suspense>
+      );
+
+      const wrapper = mount(<SuspenseComponent />);
+
+      expect(wrapper.is(SuspenseComponent)).to.equal(true);
+      expect(wrapper.find(Component)).to.have.lengthOf(1);
+      expect(wrapper.find(Fallback)).to.have.lengthOf(0);
+    });
+
+    it('can mount Suspense directly', () => {
+      const wrapper = mount(<Suspense fallback={Fallback} />);
+      expect(wrapper.is(Suspense)).to.equal(true);
+    });
+
+    it('finds fallback when given lazy component in initial mount', () => {
+      const LazyComponent = lazy(() => fakeDynamicImport(DynamicComponent));
+      const SuspenseComponent = () => (
+        <Suspense fallback={<Fallback />}>
+          <LazyComponent />
+        </Suspense>
+      );
+
+      const wrapper = mount(<SuspenseComponent />);
+
+      expect(wrapper.is(SuspenseComponent)).to.equal(true);
+      expect(wrapper.find(LazyComponent)).to.have.lengthOf(0);
+      expect(wrapper.find(Fallback)).to.have.lengthOf(1);
+    });
+
+    it('return fallback string when given lazy component in initial mount and call .debug()', () => {
+      const LazyComponent = lazy(() => fakeDynamicImport(DynamicComponent));
+      const SuspenseComponent = () => (
+        <Suspense fallback={<Fallback />}>
+          <LazyComponent />
+        </Suspense>
+      );
+
+      const wrapper = mount(<SuspenseComponent />);
+
+      expect(wrapper.debug()).to.equal(`<SuspenseComponent>
+  <Suspense fallback={{...}}>
+    <Fallback>
+      <div>
+        Fallback
+      </div>
+    </Fallback>
+  </Suspense>
+</SuspenseComponent>`);
+    });
+
+    it('return wrapped component when given loaded lazy component in initial mount', () => {
+      const LazyComponent = lazy(() => fakeDynamicImport(DynamicComponent));
+      /* eslint-disable no-underscore-dangle */
+      LazyComponent._result = DynamicComponent;
+      /* eslint-disable no-underscore-dangle */
+      LazyComponent._status = 1;
```

Comment:
also here?

Reply:
I'd like to test the case where the wrapped component is loaded here.